### PR TITLE
Profile builder frontend

### PR DESCRIPTION
**Pull request by samuelsandoval1**
## Proposed Changes
Created Front-end Profile Builder Form for Mentee and Mentors

## Types of Changes
What types of changes does your code introduce to Techish Project?
_Put an x in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation Update (if none of the other choices apply)

## Checklist
_Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x]  I have added necessary documentation (if appropriate)
- [x]  Any dependent changes have been merged and published in downstream modules

## Checklist
[Front-End Appearance](https://drive.google.com/drive/folders/1y6gsqFFt72LTAHiPmbrnxwe1VxoCrSLn?usp=sharing)

### CODE REVIEW

File: src/main/webapp/profileBuilder.html [hunk 1]
```diff
@@ -5,11 +5,137 @@
...
+    </section>
+    <!-- Footer -->
+    <footer class="py-5 bg-black">
+      <div class="container">
+        <p class="m-0 text-center text-white small">Copyright &copy; Techish 2020</p>
+      </div>
+      <!-- /.container -->
+    </footer>
+    <!-- Bootstrap core JavaScript -->
+    <script src="vendor/jquery/jquery.min.js"></script>
```
Comment (ghandre) [inline src/main/webapp/profileBuilder.html:137]: Includes in head/top?

File: src/main/webapp/ts/profileBuilder.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function setForm(value: any) {
```
Comment (ghandre) [inline src/main/webapp/ts/profileBuilder.ts:1]: Can this be  'value: string'? (L1)

File: src/main/webapp/profileBuilder.html [hunk 3]
```diff
@@ -5,11 +5,137 @@
...
+      <div class="bg-circle-4 bg-circle"></div>
+    </header>
+    <section>
+        <div class="container">
+            <div>
+                <input type="button" class="button_slide slide_down" value="Mentor" onclick="setForm(this.value)">
+                <input type="button" class="button_slide slide_down" value="Mentee" onclick="setForm(this.value)">
+            </div>
+            <form action="/profile" id="profile_form" method="POST" id="makeProfile">
+                <div>
```
Comment (ghandre) [inline src/main/webapp/profileBuilder.html:51]: remove empty div?

File: src/main/webapp/css/profileBuilder.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+.button_slide {
```
Comment (ghandre) [inline src/main/webapp/css/profileBuilder.css:2]: Assuming what's being served on appspot is up to date (as it seems to be), this renders as a grey button with a maroon border (box-shadow) with white text. Is that the intended style? The white text on grey background is hard to read.

Comment (jerrychangusa) [reply src/main/webapp/css/profileBuilder.css:2]: +1 regarding making the buttons easier to read

File: src/main/webapp/profileBuilder.html (hunk 1 above)
Comment (samuelsandoval1) [reply src/main/webapp/profileBuilder.html:137]: What do you mean by this, what line specifically?

File: src/main/webapp/ts/profileBuilder.ts (hunk 2 above)
Comment (samuelsandoval1) [reply src/main/webapp/ts/profileBuilder.ts:1]: Yea. Gotcha

Comment (samuelsandoval1) [reply src/main/webapp/ts/profileBuilder.ts]: Done

File: src/main/webapp/profileBuilder.html (hunk 3 above)
Comment (samuelsandoval1) [reply src/main/webapp/profileBuilder.html]: Done

File: src/main/webapp/css/profileBuilder.css (hunk 4 above)
Comment (samuelsandoval1) [reply src/main/webapp/css/profileBuilder.css:2]: I see your concern, in the frontend, the buttons show up as a dark grey background, with white text and maroon border. It's pretty readable on my end.

[Image Link](https://screenshot.googleplex.com/HpBuhg0tu1R.pngl)

File: src/main/webapp/ts/profileBuilder.ts (hunk 2 above)
Comment (samuelsandoval1) [reply src/main/webapp/ts/profileBuilder.ts]: Done

File: src/main/webapp/profileBuilder.html (hunk 3 above)
Comment (samuelsandoval1) [reply src/main/webapp/profileBuilder.html]: Done

File: src/main/webapp/css/profileBuilder.css (hunk 4 above)
Comment (samuelsandoval1) [reply src/main/webapp/css/profileBuilder.css:2]: Done

Comment (ghandre) [reply src/main/webapp/css/profileBuilder.css:2]: Chatted offline, looks like the version on appspot wasn't up to date + didn't see the "Front-end appearence" link in the original post; Looks good

File: src/main/webapp/profileBuilder.html (hunk 1 above)
Comment (samuelsandoval1) [reply src/main/webapp/profileBuilder.html:127]: Done

**Pull request merged**